fea-rs/src/compile/lookups/contextual.rs

```diff
@@ -72,48 +97,63 @@ impl<T> ContextualLookupBuilder<T> {
         } else {
             ChainOrNot::Context(LookupBuilder::new_with_lookups(flags, mark_set, subtables))
         };
-        (lookup, anon_lookups)
+        (lookup, finished_anon_lookups)
     }
 
-    //TODO: if we keep this, make it unwrap and ensure we always have a subtable
+    /// Returns a mutable reference to the active builder
     pub fn last_mut(&mut self) -> &mut ContextBuilder {
         self.subtables.last_mut().unwrap()
     }
 
+    /// Force a new subtable (caused by the explicit 'subtable' statement)
+    ///
+    /// For contextual lookups, a subtable break does two things: it ensures
+    /// that any subsequent contextual rules go into a new subtable (like with
+    /// other tables) but it also forces any new inline rules to go into new
+    /// lookups (e.g. lookups cannot be shared across a 'subtable break' boundary.)
```

Comment:
I wonder, is there a test that explicitly captures this condition?

Reply:
There is `ChainPosSubtable.fea` and `ChainSubstSubtable.fea` in the feaLib test suite. :)